l3kernel/doc/l3obsolete.txt

```diff
@@ -76,6 +76,12 @@ Function                            Date deprecated
 \prop_put_if_new:Nnn                     2024-03-30
 \prop_put_if_new:NnV                     2024-03-30
 \prop_put_if_new:NVn                     2024-03-30
+\regex_match:nnF                         2025-05-14
+\regex_match:nnT                         2025-05-14
+\regex_match:nnTF                        2025-05-14
+\regex_match:NnF                         2025-05-14
+\regex_match:NnT                         2025-05-14
+\regex_match:NnTF                        2025-05-14
```

Comment:
Just sharing an interesting finding 🦆 (rather than a serious suggestion): `nnF < nNF < nnT < ...`. Thus strictly speaking the order would be
```tex
\regex_match:nnF                         2025-05-14
\regex_match:NnF                         2025-05-14
\regex_match:nnT                         2025-05-14
\regex_match:NnT                         2025-05-14
\regex_match:nnTF                        2025-05-14
\regex_match:NnTF                        2025-05-14
```

Reply:
I think we've grouped the T/F/TF together in general